browser/src/control/Parts.js

```diff
@@ -122,8 +122,11 @@ L.Map.include({
 		if (docLayer._invalidatePreviews) {
 			docLayer._invalidatePreviews();
 		}
-
-		this.focus();
+		if (this._docLayer._docType === 'presentation' || this._docLayer._docType === 'drawing') {
```

Comment:
map has getter for it: `getDocType()`, please avoid using internals outside class itself if possible :)